src/main/webapp/WEB-INF/view/conversations.jsp

```diff
@@ -27,14 +28,15 @@
   <nav>
     <a id="navTitle" href="/">CodeU Chat App</a>
     <a href="/conversations">Conversations</a>
-    <% if(request.getSession().getAttribute("user") != null){ %>
+    <% Boolean isRegistered = (Boolean) request.getAttribute("is_registered");
```

Comment:
Why did you use Boolean instead of boolean? Also, I might be missing something, but do we need this "is_registered" attribute? In ConversationServlet "is_registered" is set if user is not null, so could we just use that as our check here too (so set isRegistered to (username != null) like you did in ConversationServlet)?

Reply:
I used Boolean because request.getAttribute() returns an Object whereas boolean is a primitive type. And I guess I was using isRegistered to avoid logic in the .jsp file